cms/templates/widgets/sock.html

```diff
@@ -69,28 +37,22 @@ <h3 class="title">${_("Request help with {studio_name}").format(studio_name=sett
             'text': _('edX Partner Portal'),
             'condition': is_edx_domain
         }, {
+            'href': 'https://stanfordonline.zendesk.com/hc',
+            'sr_mouseover_text': _('Access Course Staff Support on the Help Center to submit or review support tickets'),
+            'text': _('Help Center'),
+            'condition': True
+        }, {
             'href': 'https://open.edx.org',
             'sr_mouseover_text': _('Access the Open edX Portal'),
             'text': _('Open edX Portal'),
             'condition': not is_edx_domain
         }, {
-            'href': 'https://www.edx.org/course/overview-creating-edx-course-edx-edx101#.VO4eaLPF-n1',
```

Comment:
We already had this link on our instance. It's a free, self-paced course about course creation.
Any particular reason to remove it?